core/models/data_declaration.py

```diff
@@ -293,7 +293,9 @@ def to_dict(self):
             "de_identification": self.deidentification_method.name
             if self.deidentification_method
             else None,
-            "consent_status": self.consent_status.name if self.consent_status else None,
+            "consent_status": self.consent_status.name
+            if self.consent_status != ConsentStatus.unknown
+            else None,
```

Comment:
This means that the exported data regarding data declarations will not contain the unknown consent status anymore. Is it the intended behavior?

Reply:
Thanks @Fancien for noticing. This is correct behaviour as the json schemas do not allow "unknown" value. They never had as a matter of fact :)